docs/4.3/enterprise/workflow/ssh_approval_slack.md

```diff
@@ -0,0 +1,223 @@
+## Installing the Teleport Slack Plugin
+To start using Teleport Plugins, you will need the teleport-slackbot executable.  See the [README](README.md) for building the teleport-slackbot executable in the Setup section.  Place the executable in the appropriate /usr/bin or /usr/local/bin on the server installation.
+
+### Configuration File
+Teleport Slack plugin has its own config file in TOML format. Before starting the plugin, you'll need to generate (or just copy the one below) and edit that config.
+
+To generate a config file, you can do this: 
+```bash
+teleport-slackbot configure > /etc/teleport-slackbot.toml
+```
+
+Note that it saves the config file to `/etc/teleport-slackbot.toml`. You'll be able to point the plugin to any config file path you want, but it'll pick up `/etc/teleport-slackbot.toml` by default. 
+
+#### Editing the config file
+In the Teleport section, use the certificates you've generated with `tctl auth sign` before. The plugin installer creates a folder for those certificates in `/var/lib/teleport/plugins/slackbot/` — so just move the certificates there and make sure the config points to them. 
+
+In Slack section, use the OAuth token, signing token, setup the desired channel name. The listen URL is the URL the plugin will listen for Slack callbacks. 
+
+Then set the plugin callback (where Slack sends its requests) to an address you like, and provide the TLS certificates for that http server to use. 
+
+By default, Teleport Slack plugin will run with TLS on. You can override it with `--insecure-no-tls` flag, but that's not encouraged.  
+
+```TOML
+# Example Teleport Slack Plugin config file
+[teleport]
+auth-server = "example.com:3025"  # Teleport Auth Server GRPC API address
+client-key = "/var/lib/teleport/plugins/slackbot/auth.key" # Teleport GRPC client secret key
+client-crt = "/var/lib/teleport/plugins/slackbot/auth.crt" # Teleport GRPC client certificate
+root-cas = "/var/lib/teleport/plugins/slackbot/auth.cas"   # Teleport cluster CA certs
+
+[slack]
+token = "api-token"       # Slack Bot OAuth token
+secret = "signing-secret-value"   # Slack API Signing Secret
+channel = "channel-name"  # Slack Channel name to post requests to
+
+[http]
+listen = ":8081"          # Slack interaction callback listener port
+# https-key-file = "/var/lib/teleport/plugins/slackbot/server.key"  # TLS private key
+# https-cert-file = "/var/lib/teleport/plugins/slackbot/server.crt" # TLS certificate
+
+[log]
+output = "stderr" # Logger output. Could be "stdout", "stderr" or "/var/lib/teleport/slackbot.log"
+severity = "INFO" # Logger severity. Could be "INFO", "ERROR", "DEBUG" or "WARN".
+```
+
+## Test Run 
+
+Assuming that Teleport is running, and you've created the Slack app, the plugin config, and provided all the certificates — you can now run the plugin and test the workflow!
+
+`teleport-slackbot start`
+
+If everything works fine, the log output should look like this:
+
+```bash
+vm0:~/slackbot sudo ./teleport=slackbot start
```

Comment:
We should be able to run these plugins without needing to be `root` for security

Reply:
Created an issue here https://github.com/gravitational/teleport-plugins/issues/93